scipy/cluster/hierarchy.py

```diff
@@ -2349,7 +2399,7 @@ def llf(id):
         count_sort=count_sort,
         distance_sort=distance_sort,
         show_leaf_counts=show_leaf_counts,
-        i=2*n - 2,
+        i=2 * n - 2,
```

Comment:
This was changed in PEP8 a few years ago; there should be spaces only around the operators with lower precedence. So the way it was before you changed it is better. Can you revert this change?

See https://www.python.org/dev/peps/pep-0008/#other-recommendations